kasa/discover.py

```diff
@@ -34,10 +34,12 @@ def __init__(
         target: str = "255.255.255.255",
         timeout: int = 5,
         discovery_packets: int = 3,
+        interface: bytes = b""
```

Comment:
This should be a string that defaults to `None` and use `encode()` later (if the string is not really accepted by `setsockopt`, but I think it should be?).

Reply:
Done